addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/CometVisuDashboardTile.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.ui.cometvisu.internal;
+
+import org.openhab.ui.dashboard.DashboardTile;
+
+public class CometVisuDashboardTile implements DashboardTile {
+    @Override
+    public String getName() {
+        return "CometVisu";
+    }
+
+    @Override
+    public String getUrl() {
+        return "../cometvisu/?config=demo";
+    }
+
+    @Override
+    public String getOverlay() {
+        return "html5";
+    }
+
+    @Override
+    public String getImageUrl() {
+        return "img/cometvisu.png";
```

Comment:
since you are serving yourself content, you could move the png file there (i.e. have a url like ../cometvisu/dashboard.png), so that the png does not have to be added in the ui.dashboard bundle at all (which should only be a last resort...).